packages/amplify-category-storage/src/provider-utils/awscloudformation/index.ts

```diff
@@ -10,6 +10,8 @@ import { importedS3EnvInit, importS3 } from './import/import-s3';
 
 export { importResource } from './import';
 
+const path = require('path');
```

Comment:
```suggestion
import * as path from 'path';
```

Reply:
I've made this adjustment as well and squashed the commits into one tidy commit

Reply:
When I get time I'll also try moving the helper functions if that's possible in a neat way and submit a PR for you to check